src/tests/functional/shared_test_classes/src/subgraph/rotary_pos_emb.cpp

```diff
@@ -61,8 +61,8 @@ std::shared_ptr<ov::Model> RoPETestLlama2StridedSlice::buildROPE_Llama2(int batc
                                                          {"new_axis_mask", {}},
                                                          {"shrink_axis_mask", {}},
                                                          {"ellipsis_mask", {}}});
-    auto squeeze_Squeeze = makeOP<ov::op::v0::Squeeze>({slice_Slice, 1});
-    auto squeeze_Squeeze_435 = makeOP<ov::op::v0::Squeeze>({squeeze_Squeeze, 0});
+    auto squeeze_Squeeze = makeOP<ov::op::v0::Squeeze>({slice_Slice, 1}, {{"pytorch_dynamic_rank", false}});
```

Comment:
is it really required to provide this attribute in all transformations?

Reply:
Fixed